plugin-cgroupv2/src/k8s/plugin.rs

```diff
@@ -61,39 +68,40 @@ impl AlumetPlugin for K8sPlugin {
         }))
     }
 
-    fn start(&mut self, alumet: &mut AlumetPluginStart) -> anyhow::Result<()> {
-        let v2_used: bool = cgroup_v2::is_accessible_dir(&PathBuf::from("/sys/fs/cgroup/"));
+    fn start(&mut self, alumet: &mut alumet::plugin::AlumetPluginStart) -> anyhow::Result<()> {
```

Comment:
Why not importing it like the others ?
```suggestion
    fn start(&mut self, alumet: &mut AlumetPluginStart) -> anyhow::Result<()> {
```

Reply:
passed through my verification